client/src/redux/library/libraryReducers.js

```diff
@@ -57,7 +58,8 @@ export const libraryReducer = (state = initialState, { type, payload }) => {
     case libraryTypes.SEARCH:
       return {
         ...state,
-        docs: payload.docs,
+        docs: payload.useDocsForDropdown ? state.docs : payload.docs,
+        docsForDropdown: payload.useDocsForDropdown ? payload.docs : state.docs,
```

Comment:
@dzimmerman902 I came across a situation where I needed two lists of library documents at the same time, so I added a second list here that could be used. Let me know if you have any thoughts on the execution of this! The other relevant piece of this is in the `client/src/redux/library/actions/search.js` file.